Assets/Scripts/Data/Pickup Pup/PPDataController.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/*
```

Comment:
I'm getting an IO Exception whenever I run the game in editor:

"EndOfStreamException: Failed to read past end of stream."